packages/content-item-app/src/app/item-page/item-page.component.ts

```diff
@@ -13,27 +13,19 @@ import {
   templateUrl: './item-page.component.html',
   styleUrls: ['./item-page.component.scss']
 })
-export class ItemPageComponent implements OnInit, PluginComponent {
-  value1: number = 2;
+export class ItemPageComponent implements PluginComponent {
+  value1 = 2;
   appEventPub: TopicPublisher<Message>;
 
-  constructor() {
-  }
-
-  ngOnInit(): void {
-  }
-
-
   initialize(context: PluginContext): void {
-    this.appEventPub = context.publisher('app:event');
+    this.appEventPub = context.messaging.createPublisher('app:event');
```

Comment:
I need to look at this one, I have changed it on Friday and maybe I missed something in your PR. I made it so on the application layer we should not be creating publishers, they should be created on demand. 

Once you doSend, I check if hte channel is create if not I will create it . The same applies for subscribers, I want to do .onMessage (or newly .subscribe) it shoudl create new channel if it does nto exists. To easy its usually and not to worry about Publishers and subcribers and handle this internally

Reply:
Didn't yet refactor this, so it works as you have coded it initially

Reply:
This is the think I overlooked it. My latest changes I was about to do on this demo app is with updated API that I did on friday.. 

I mean when I refactor this this will work as well right, cuz I would update this part too. I have deprecated these method and favor only .publish / .subscribe . Even this older code should work. 

Or any specific reason why `dont refactor it yet ` ?